contracts/ERC20MultiDelegate.sol

```diff
@@ -189,25 +215,25 @@ contract ERC20MultiDelegate is ERC1155, Ownable {
         return proxyAddress;
     }
 
-    function getBalanceForDelegate(
+    function _getBalanceForDelegate(
         address delegate
     ) internal view returns (uint256) {
-        return ERC1155(this).balanceOf(msg.sender, uint256(uint160(delegate)));
+        return balanceOf(msg.sender, uint256(uint160(delegate)));
     }
 
-    function retrieveProxyContractAddress(
+    function _retrieveProxyContractAddress(
         ERC20Votes _token,
         address _delegate
     ) private view returns (address) {
-        bytes memory bytecode = abi.encodePacked(
-            type(ERC20ProxyDelegator).creationCode, 
+        bytes memory bytecode = bytes.concat(
+            type(ERC20ProxyDelegator).creationCode,
             abi.encode(_token, _delegate)
         );
         bytes32 hash = keccak256(
-            abi.encodePacked(
```

Comment:
`abi.encodePacked` is actually fine here, I think.

Reply:
reverted